draft-ietf-privacypass-auth-scheme.md

```diff
@@ -403,22 +406,45 @@ the Authorization header field as follows:
 Authorization: PrivateToken token="abc..."
 ~~~
 
-For token types that support public verifiability, origins verify the token
-authenticator using the public key of the issuer, and validate that the signed
-message matches the concatenation of the client nonce and the hash of a
-valid TokenChallenge. For context-bound tokens, origins store or reconstruct
-the contexts of previous TokenChallenge structures in order to validate the
-token. A TokenChallenge MAY be bound to a specific TLS session with a client,
-but origins can also accept tokens for valid challenges in new sessions.
-Origins SHOULD implement some form of double-spend prevention that prevents
-a token with the same nonce from being redeemed twice. This prevents clients
-from "replaying" tokens for previous challenges. For context-bound tokens,
-this double-spend prevention can require no state or minimal state, since
-the context can be used to verify token uniqueness.
+For context-bound tokens, origins store or reconstruct the contexts of previous
+TokenChallenge structures in order to validate the token. A TokenChallenge MAY
```

Comment:
Now that we're going back to this, does this need to be a normative MAY, or can it be:
```suggestion
TokenChallenge structures in order to validate the token. A TokenChallenge can
```

Reply:
This was existing text, so I'm fine relaxing it.